fabtests/ubertest/uber.c

```diff
@@ -596,6 +596,15 @@ static int ft_server_child()
 	if (ret && ret != -FI_ENODATA) {
 		FT_PRINTERR("fi_getinfo", ret);
 	} else {
+		/* Temporary fix to run one set of tests, rather than
+		 * iterating over all interfaces / addresses.
+		 * TODO: Remove iteration from ft_fw_process_list_server.
+		 */
+		if (info->next) {
+			fi_freeinfo(info->next);
+			info->next = NULL;
+		}
+
```

Comment:
I think we're going to need the same hack before ft_fw_process_list_client since the client also iterates over its interfaces

Reply:
bah - updating - thanks